src/sqlparse.yrl

```diff
@@ -417,13 +420,13 @@ plsql_body -> BEGIN statement_pragma_list END : {'plsql_body', '$2'}.
 statement_pragma_list ->                       statement_pragma :         ['$1'].
 statement_pragma_list -> statement_pragma_list statement_pragma : '$1' ++ ['$2'].
 
-statement_pragma -> assign_statement : '$1'.
+statement_pragma -> assignment_statement : '$1'.
 
-assign_statement -> parameter ':=' scalar_opt_as_exp ';' : {':=', '$1', '$3'}.
+assignment_statement -> parameter ':=' scalar_opt_as_exp ';' : {':=', '$1', '$3'}.
 
 procedure_call -> BEGIN function_ref_list END : {'begin procedure', '$2'}.
 procedure_call -> BEGIN sql_list          END : {'begin procedure', '$2'}.
-procedure_call -> CALL  function_ref          : {'call procedure',  '$2'}.
+procedure_call -> CALL  function_ref          : {'call procedure', '$2'}.

```

Comment:
Original alignment(probably introduced by a previous PR) was correct!

Reply:
Honestly I have problems with right and wrong alignment - there are no rules (at least defined) and in the end it's all about personal taste regarding sorting and symmetric alignment.

Reply:
It is now changed back.